.github/ISSUE_TEMPLATE/sync_request.md

```diff
@@ -6,7 +6,7 @@ title: Issue for - Sync with Upstream v1.2x
 ---
 
 ## What would you like to be added
-Gardener autoscaler should be synced with [Kubernetes Autoscaler 1.2x](https://github.com/kubernetes/autoscaler/releases/tag/cluster-autoscaler-1.2x.y)
+Gardener autoscaler should be synced with respecrtive [Kubernetes Autoscaler 1.x.y Release](https://github.com/kubernetes/autoscaler/releases/)
```

Comment:
```suggestion
Gardener autoscaler should be synced with respective [Kubernetes Autoscaler 1.x.y Release](https://github.com/kubernetes/autoscaler/releases/)
```

Reply:
The title still has `v1.2x` . Kindly fix that.

Reply:
fixed.